plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/GoogleLoginService.java

```diff
@@ -148,10 +146,7 @@ public boolean hasAccounts() {
   @Override
   public Set<Account> getAccounts() {
     synchronized (loginState) {
-      // 'accounts' is a reference to a copy of Accounts maintained in 'loginState'.
-      // ('loginState.listAccounts()' returns a copy.) We intend to return this
-      // reference to callers, while never modifying the set itself.
-      return accounts;
+      return new HashSet<Account>(accounts);
```

Comment:
Is the comment at the `accounts` field declaration still valid in light of this change? If so, I'm not fully understanding what it means.